--- wordpress/wp-admin/setup-config.php~	2010-09-08 12:29:43.000000000 +0300
+++ wordpress/wp-admin/setup-config.php	2010-09-08 15:01:48.112464938 +0300
@@ -51,7 +51,7 @@
 $configFile = file(ABSPATH . 'wp-config-sample.php');
 
 // Check if wp-config.php has been created
-if (file_exists(ABSPATH . 'wp-config.php'))
+if (file_exists(ABSPATH . 'wp-config.php') && filesize(ABSPATH . 'wp-config.php'))
 	wp_die("<p>The file 'wp-config.php' already exists. If you need to reset any of the configuration items in this file, please delete it first. You may try <a href='install.php'>installing now</a>.</p>");
 
 // Check if wp-config.php exists above the root directory but is not part of another install
