--- wordpress-3.5.1/wp-admin/setup-config.php~	2013-02-06 16:10:02.000000000 +0200
+++ wordpress-3.5.1/wp-admin/setup-config.php	2013-02-06 16:11:57.583090884 +0200
@@ -68,7 +68,7 @@
 $config_file = file(ABSPATH . 'wp-config-sample.php');
 
 // Check if wp-config.php has been created
-if ( file_exists( ABSPATH . 'wp-config.php' ) )
+if ( file_exists( ABSPATH . 'wp-config.php' ) && filesize( ABSPATH . 'wp-config.php' ) )
 	wp_die( '<p>' . sprintf( __( "The file 'wp-config.php' already exists. If you need to reset any of the configuration items in this file, please delete it first. You may try <a href='%s'>installing now</a>." ), 'install.php' ) . '</p>' );
 
 // Check if wp-config.php exists above the root directory but is not part of another install
