--- wordpress-3.0.1/wp-config-sample.php	2010-09-08 18:03:29.421631471 +0300
+++ wordpress-3.0.1/wp-config-sample.php	2010-09-08 18:28:37.181139000 +0300
@@ -80,6 +80,12 @@
  */
 define('WP_DEBUG', false);
 
+/**
+ * Options for setting up Multisite.
+ * @see http://codex.wordpress.org/Create_A_Network
+ */
+define('WP_ALLOW_MULTISITE', false);
+
 /* That's all, stop editing! Happy blogging. */
 
 /** Absolute path to the WordPress directory. */
--- wordpress-3.5.1/wp-includes/ms-default-constants.php~	2013-02-06 16:12:50.000000000 +0200
+++ wordpress-3.5.1/wp-includes/ms-default-constants.php	2013-02-06 16:14:43.844078357 +0200
@@ -26,7 +26,7 @@
 
 	// Base uploads dir relative to ABSPATH
 	if ( !defined( 'UPLOADBLOGSDIR' ) )
-		define( 'UPLOADBLOGSDIR', 'wp-content/blogs.dir' );
+		define( 'UPLOADBLOGSDIR', '/var/lib/wordpress' );
 
 	// Note, the main site in a post-MU network uses wp-content/uploads.
 	// This is handled in wp_upload_dir() by ignoring UPLOADS for this case.
@@ -34,8 +34,8 @@
 		define( 'UPLOADS', UPLOADBLOGSDIR . "/{$wpdb->blogid}/files/" );
 
 		// Uploads dir relative to ABSPATH
-		if ( 'wp-content/blogs.dir' == UPLOADBLOGSDIR && ! defined( 'BLOGUPLOADDIR' ) )
-			define( 'BLOGUPLOADDIR', WP_CONTENT_DIR . "/blogs.dir/{$wpdb->blogid}/files/" );
+		if ( ('wp-content/blogs.dir' == UPLOADBLOGSDIR  || '/var/lib/wordpress' == UPLOADBLOGSDIR) && ! defined( 'BLOGUPLOADDIR' ) )
+			define( 'BLOGUPLOADDIR', "/var/lib/wordpress/{$wpdb->blogid}/files/" );
 	}
 }
 
