--- wordpress/wp-includes/pluggable.php~	2010-09-09 16:45:23.000000000 +0300
+++ wordpress/wp-includes/pluggable.php	2010-09-09 18:51:10.849503438 +0300
@@ -279,8 +279,7 @@
 
 	// (Re)create it, if it's gone missing
 	if ( !is_object( $phpmailer ) || !is_a( $phpmailer, 'PHPMailer' ) ) {
-		require_once ABSPATH . WPINC . '/class-phpmailer.php';
-		require_once ABSPATH . WPINC . '/class-smtp.php';
+		require_once 'class.phpmailer.php';
 		$phpmailer = new PHPMailer();
 	}
 
