--- wordpress-3.5.1/wp-includes/script-loader.php~	2013-02-06 16:23:21.000000000 +0200
+++ wordpress-3.5.1/wp-includes/script-loader.php	2013-02-06 16:24:06.019821611 +0200
@@ -192,7 +192,7 @@
 
 	$scripts->add( 'jcrop', "/wp-includes/js/jcrop/jquery.Jcrop.min.js", array('jquery'), '0.9.10');
 
-	$scripts->add( 'swfobject', "/wp-includes/js/swfobject.js", array(), '2.2-20120417');
+	$scripts->add( 'swfobject', "/js/swfobject.js", array(), '2.2-20120417');
 
 	// common bits for both uploaders
 	$max_upload_size = ( (int) ( $max_up = @ini_get('upload_max_filesize') ) < (int) ( $max_post = @ini_get('post_max_size') ) ) ? $max_up : $max_post;
