--- wordpress/wp-load.php~	2009-03-20 10:38:16.000000000 +0200
+++ wordpress/wp-load.php	2009-03-20 10:40:16.081599269 +0200
@@ -21,7 +21,12 @@
 
 error_reporting(E_ALL ^ E_NOTICE ^ E_USER_NOTICE);
 
-if ( file_exists( ABSPATH . 'wp-config.php') ) {
+if ( file_exists( '/etc/webapps/wordpress/wp-config.php') ) {
+
+	/** The config file resides in WEBAPPS */
+	require_once( '/etc/webapps/wordpress/wp-config.php' );
+
+} elseif ( file_exists( ABSPATH . 'wp-config.php') ) {
 
 	/** The config file resides in ABSPATH */
 	require_once( ABSPATH . 'wp-config.php' );
--- wordpress-3.5.1/wp-admin/setup-config.php~	2013-02-06 16:21:34.000000000 +0200
+++ wordpress-3.5.1/wp-admin/setup-config.php	2013-02-06 16:22:14.098004158 +0200
@@ -275,7 +275,7 @@
 </script>
 <?php
 	else :
-		$handle = fopen(ABSPATH . 'wp-config.php', 'w');
+		$handle = fopen(ABSPATH . '/etc/webapps/wordpress/wp-config.php', 'w');
 		foreach( $config_file as $line ) {
 			fwrite($handle, $line);
 		}
--- wordpress-3.0.1/wp-config.php~	2010-09-08 12:02:37.000000000 +0300
+++ wordpress-3.0.1/wp-config.php	2010-09-08 12:05:20.193097026 +0300
@@ -16,13 +16,13 @@
 
 // ** MySQL settings - You can get this info from your web host ** //
 /** The name of the database for WordPress */
-define('DB_NAME', 'database_name_here');
+define('DB_NAME', 'wordpress');
 
 /** MySQL database username */
-define('DB_USER', 'username_here');
+define('DB_USER', 'mysql');
 
 /** MySQL database password */
-define('DB_PASSWORD', 'password_here');
+define('DB_PASSWORD', '');
 
 /** MySQL hostname */
 define('DB_HOST', 'localhost');
